packages/components/src/utils/element-rect.ts

```diff
@@ -125,6 +125,11 @@ export type ElementOffsetRect = {
 	 * the element.
 	 */
 	left: number;
+	/**
+	 * The distance from the right edge of the offset parent to the right edge of
+	 * the element.
+	 */
+	right: number;
```

Comment:
In this PR, the `right` value is calculated in the `getElementOffsetRect()` function, but if the `right` value is inappropriate for the `getElementOffsetRect()` function itself, we can also calculate the `right` value directly in the `Tabs` component.